tests/posix/env/prj.conf

```diff
@@ -1,3 +1,6 @@
 CONFIG_ZTEST=y
 CONFIG_POSIX_ENV=y
+# Let's explicitly choose PICOLIBC, so it is used if supported even if it would not have been the
+# default (otherwise native targets default to the host C library)
+CONFIG_PICOLIBC=y
```

Comment:
If this is for native targets only, maybe put this into `boards/native_targets.conf` and create `boards/native_sim.conf`, ... to include `native_targets.conf`?

Reply:
.conf files cannot include other .conf files. And as mentioned here https://github.com/zephyrproject-rtos/zephyr/pull/72917#discussion_r1604865520 there is quite many native targets, so we end up with a lot of files.

Reply:
Fair enough